src/App.tsx

```diff
@@ -2,15 +2,17 @@ import React from 'react';
 import Header from './components/Header'
 import Card from './components/Card'
 import Footer from './components/Footer'
-import items from './assets/items'
+import { items }from './assets/items'
 
 import './css/skeleton.css'
 import './css/normalize.css'
+import MusicPlayer from './components/MusicPlayer';
 
 function App(): JSX.Element {
+    const [view, setView] = React.useState(false)
```

Comment:
Let's make it more clear.

If we are using view, then its type should be an enum value instead of a boolean because we want to make the view more declarative. 

Here, we are just using the state if we want to show music player or not and hide or show it. We can create a custom hook named `useVisibilityToggle` for it and use it here

```
const useVisibilityToggle = (initialState: boolean): {
  isVisible: boolean;
  show: () => void;
  hide: () => void;
  toggleVisibility: () => void
} => {
  const [isVisible, setIsVisible] = React.useState(initialState);

  const show = useCallback(() => {
    setIsVisible(true);
  }, []);

    const hide = useCallback(() => {
      setIsVisible(true);
    }, []);

   const toggleVisibility = useCallback(() => {
    setIsVisible(visible => !visible);
   }, []);

  return { isVisible, show, hide, toggleVisibility }
}
```